docs/src/documentation/02-foundation/10-typography/01-general.mdx

```diff
@@ -21,22 +21,24 @@ To stand out from the page, we use <InlineToken name="colorHeading" /> as the **
 
 ### Heading font sizes
 
-| Type                | Size                                                  |
-| ------------------- | ----------------------------------------------------- |
-| Display Title       | <InlineToken name="fontSizeHeadingDisplay" />         |
-| Display Subtitle    | <InlineToken name="fontSizeHeadingDisplaySubtitle" /> |
-| Title 1             | <InlineToken name="fontSizeHeadingTitle1" />          |
-| Title 2             | <InlineToken name="fontSizeHeadingTitle2" />          |
-| Title 3             | <InlineToken name="fontSizeHeadingTitle3" />          |
-| Title 4             | <InlineToken name="fontSizeHeadingTitle4" />          |
-| Title 5 (Uppercase) | <InlineToken name="fontSizeHeadingTitle5" />          |
+| Type             | Size                                                  |
+| ---------------- | ----------------------------------------------------- |
+| Display Title    | <InlineToken name="headingDisplayFontSize" />         |
+| Display Subtitle | <InlineToken name="headingDisplaySubtitleFontSize" /> |
+| Title 0          | <InlineToken name="headingTitle0FontSize" />          |
+| Title 1          | <InlineToken name="headingTitle1FontSize" />          |
+| Title 2          | <InlineToken name="headingTitle2FontSize" />          |
+| Title 3          | <InlineToken name="headingTitle3FontSize" />          |
+| Title 4          | <InlineToken name="headingTitle4FontSize" />          |
+| Title 5          | <InlineToken name="headingTitle5FontSize" />          |
+| Title 6          | <InlineToken name="headingTitle6FontSize" />          |
 
 ### Basic guidelines for headings
 
 <Guideline type="do" title="Be respectful of the content hierarchy">
 
 Try not to skip heading levels.
-If you've used Title 1, the next level used, if intentionally smaller, should be Title 2.
+If you've used level 1, the next level used, if intentionally smaller, should be level 2.
```

Comment:
Not sure if we should use level 0 instead. I guess one can get the point in any case.

Reply:
I think `level` is actually the correct terminology. We can have any type we want, in any order, but the level (h1 > h2 > h3) is the one that should be respected. While the prop `type` refers to style only, the prop `level` refers to the actual heading level, and that is where the order matters, imo